src/System.Net.NameResolution/tests/PalTests/System.Net.NameResolution.Pal.Tests.msbuild

```diff
@@ -133,9 +133,6 @@
     <Compile Include="$(CommonPath)\Interop\Unix\Interop.Libraries.cs">
       <Link>Interop\Unix\Interop.Libraries.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Unix\libc\Interop.close.cs">
-      <Link>Interop\Unix\libc\Interop.close.cs</Link>
-    </Compile>
```

Comment:
... but the windows build fails and it's needed for the same source line that changed being pulled into other projects. Fixed. :blush: